### PR TITLE
how to modify config.json when using docker

### PR DESCRIPTION
**Pull request by bayethiernodiop**

### CODE REVIEW

**Comment by mritunjaygoutam12** [comment]
thank you.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -52,6 +52,8 @@ This is going to:
 
 You can visit the website at [http://localhost:9000](http://localhost:9000).
 
+TO modify the configuration listed in config-helper.ts (e.g server port) you need to modify the file `local-docker-config.json` in `voice-web/docker` and not create a `config.json` in the root of the project.
```
Comment (mritunjaygoutam12) [inline CONTRIBUTING.md]: small 'O' need at 'T0'. should we need to tell "not create a `config.json` " here ?

Comment (bayethiernodiop) [reply CONTRIBUTING.md]: oh sorry. you right

Comment (bayethiernodiop) [reply CONTRIBUTING.md]: done

**Comment by Gregoor** [comment]
Thanks for fixing the docs!

**Pull request merged**